src/fsharp/symbols/Symbols.fsi

```diff
@@ -721,6 +721,15 @@ type FSharpMemberOrFunctionOrValue =
     internal new: SymbolEnv * ValRef -> FSharpMemberOrFunctionOrValue
     internal new: SymbolEnv * Infos.MethInfo -> FSharpMemberOrFunctionOrValue
 
+    /// Indicates if this is a top level function parameter (like x in let f x = ... or x in fun x -> ...)
```

Comment:
this comment and the one below it are identical. Is that intended?

Reply:
Not really, it's more about not knowing what to write as there is lot's of duplication of that property

Reply:
Oh, no. This in particular is my mistake.